docs/gen-client-docs.py

```diff
@@ -275,6 +296,23 @@ def _to_typed_args(argspec: inspect.FullArgSpec, required: bool) -> list[TypedAr
             out.append(TypedArgument(name=arg, tpe=tpe, default=defaults.get(arg)))
         return out
 
+    def class_properties(self, inst) -> list[PropertyDoc]:
+        property_docs = []
+        for name in dir(inst):
+            if name[0] == '_':
+                # private members
+                continue
+            instance_attr = getattr(inst, name)
+            if name.startswith('_'):
+                continue
+            if inspect.ismethod(instance_attr):
```

Comment:
Can we try this? It is probably a bit more robust.
```suggestion
            if not isinstance(inst, property):
```

Reply:
From a reflection point of view, the exact object we have here is not a property.